src/material/core/typography/_typography-utils.scss

```diff
@@ -96,3 +97,9 @@
   @include font-shorthand($font-size, $font-weight, $line-height, $font-family);
   letter-spacing: letter-spacing($config, $level);
 }
+
+/// Coerce a value to `em` if it is a unitless number, otherwise returns
+/// the value provided.
+@function private-coerce-unitless-to-em($value) {
+  @return if(math.is-unitless($value), 1em * $value, value);
```

Comment:
I think you're missing a `$` on the last `value`.

Reply:
Good catch, fixed